impl/src/main/java/com/okta/sdk/impl/client/DefaultClientBuilder.java

```diff
@@ -210,6 +206,28 @@ else if (SYSPROPS_TOKEN.equalsIgnoreCase(location)) {
             clientConfig.setProxyPassword(props.get(DEFAULT_CLIENT_PROXY_PASSWORD_PROPERTY_NAME));
         }
 
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME))) {
+            clientConfig.setAuthorizationMode(props.get(DEFAULT_CLIENT_AUTHORIZATION_MODE_PROPERTY_NAME));
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME))) {
+            clientConfig.setClientId(props.get(DEFAULT_CLIENT_CLIENT_ID_PROPERTY_NAME));
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_SCOPES_PROPERTY_NAME))) {
+            List<String> scopes = new ArrayList<>();
+            scopes.add(props.get(DEFAULT_CLIENT_SCOPES_PROPERTY_NAME));
+            clientConfig.setScopes(scopes);
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_PRIVATE_KEY_KEY_FILE_PATH_PROPERTY_NAME))) {
+            clientConfig.setKeyFilePath(props.get(DEFAULT_CLIENT_PRIVATE_KEY_KEY_FILE_PATH_PROPERTY_NAME));
+        }
+
+        if (Strings.hasText(props.get(DEFAULT_CLIENT_PRIVATE_KEY_ALGORITHM_PROPERTY_NAME))) {
+            clientConfig.setAlgorithm(props.get(DEFAULT_CLIENT_PRIVATE_KEY_ALGORITHM_PROPERTY_NAME));
+        }
+
```

Comment:
Same as prior about `keyFile` and `algorithm`

Reply:
Ack again :)